src/templates/content/next_steps.html

```diff
@@ -0,0 +1,26 @@
+<div class="card">
+    <div class="card-container">
+        <div id="card-header">
+            <h4><b>Congratulations!</b></h4>
+        </div>
+        <div class="card-content">
+            <p>You've deployed your app locally. Now what?</p>
+            <ul>
+                <li>Check out Hot-reloading</li>
+                    <ul>
+                        <li>Go to <kbd class="cmp_input">flask/src/templates/base.html</kbd> in your project folder and open it.</li>
+                        <li>Update the subtitle <kbd class="cmp_input">“Favorite Movies”</kbd> to <kbd class="cmp_input">“&lt;Your name&gt;’s Favorite Movies”</kbd></li>
+                        <li>The app will automatically apply the new changes when the page refreshes!</li>
```

Comment:
Does hot reloading still work with this app?

Reply:
As close as I could get too hot reloading w/ flask. The page will change after you refresh, but won't automatically refresh on code change.

Reply:
Just leaving a note here to remove the section about hot reloading since it doesn't apply here, then I'll approve